### PR TITLE
Add pencil to bonuses

### PR DESCRIPTION
**Pull request by rastorc3v**

### CODE REVIEW

File: src/app/features/add-bonus/add-bonus.component.ts [hunk 1]
```diff
@@ -29,13 +30,15 @@ export class AddBonusComponent implements OnInit, OnDestroy {
   public isFormActive = false;
   public bonusFormConfig: IBonusFormConfig;
   public bonusId: string;
-  public bonusButtonLabel = 'Apply';
+  public isForm: boolean;
+  public bonusButtonLabel = '<img class="update-img" src="assets/images/pencil.png" alt="update bonus" /><span>Edit</span>';
```
Comment (rostic54) [inline src/app/features/add-bonus/add-bonus.component.ts:34]: It's really scared solution, better create config object, with isForm and bunusButtonLabel

Comment (rastorc3v) [reply src/app/features/add-bonus/add-bonus.component.ts:34]: I'm sorry, it's my mistake. isForm field isn't needed here

**Review by umilanovich — APPROVED** [review]

File: src/app/features/add-bonus/add-bonus.component.ts (hunk 1 above)
Comment (rostic54) [reply src/app/features/add-bonus/add-bonus.component.ts:33]: I mean keep bonus btn like html, you can pass template

Comment (rastorc3v) [reply src/app/features/add-bonus/add-bonus.component.ts:33]: What's difference between html and template? I misunderstood

**Pull request merged**